front/templates/front/decks.html

```diff
@@ -48,6 +48,14 @@ <h5>Title description, Sep 2, 2017</h5>
   {% if user.is_authenticated %}
     {{ block.super }}
     <!-- scripts - decks.html -->
+    <script>
+      function generate_api_v1_deck(){
```

Comment:
I think it is fine solution, but make it more generic and extract it to another file, so you could use this in another view.

Reply:
ok i do it in other way